portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm

```diff
@@ -164,8 +164,10 @@
 
         // put the focus back on the portal wrapper so that keyboard navigation works
         setTimeout(function() {
-
-            document.querySelector('.Mrphs-portalWrapper').focus();
+            // Do not mess with the focus if the login fields are in focus right now
```

Comment:
If the comment is required here, Can we do something like this?
```suggestion
            // Preserve focus on login fields if they're currently active
```

Reply:
Of course :)